webui/react/src/pages/FlatRuns/columns.ts

```diff
@@ -60,9 +61,9 @@ const EXCLUDED_SEARCH_DEFAULT_COLUMNS: RunColumn[] = [
 
 export type RunColumn = (typeof runColumns)[number];
 
-export const defaultRunColumns: RunColumn[] = [...runColumns];
+export const defaultRunColumns: RunColumn[] = runColumns.slice(0, -1);
```

Comment:
nit: might be worth explicitly filtering out "archived" to avoid issues if the array gets reordered